questions/hostilefriendly.md

```diff
@@ -0,0 +1,66 @@
+# How to make hostile mobs friendly? / disable PvP?
+
+## Dissable both hostile and PvP
+This method can be used to make both, dissable PvP and make hostile mobs passive.
+
+### Teams (Java only)
+In java, we can create teams, and we can configure friendly fire to make people of that team unable to attack each other. This also works for mobs. In this example we are going to make zombies passive to players.
+
+    # in chat / load function
+    team add Friendly
+    team join @a friendly
+    
+    # tick function / command blocks
+    team join @e[team=!friendly,type=zombie] friendly
+
+
+### Weakness and resistance (Java and Bedrock)
+If you can’t use the `/team` command (because you are using it for another thing or you are in bedrock) you can use effects. If we give resistance level 5 or higher the entity will be invulnerable to all damages except the `/kill` command. Weakness is recommended to avoid the player or mob cause knockback.
+
+> [!NOTE]
+> The entity will be invulnerable even to things as fall damage or entity craming, attacks that aren’t caused by the player
```

Comment:
I'd change this to
"The entity will be invulnerable even to other damage sources, such as fall damage, entity cramming or attacks that aren’t caused by the player"